src/assets/Enums.ts

```diff
@@ -123,7 +123,7 @@ export const COIF_EXPIRY_TEST_TYPES = {
 };
 
 export const LEC_TEST_TYPES = {
-  IDS: ["39", "44", "45"]
+  IDS: ["44", "45", "200", "201"]
```

Comment:
I would suggest to add these numbers into constants with some explanation. For example
`export const HEAVY_VEHICLE_TEST_ID = "44"`

Reply:
Thanks Suleiman, it's a good suggestion and busting those out into more descriptive constants would certainly make the code more readable. 

Currently these changes are consistent with what is already there, ie COIF_EXPIRY_TEST_TYPES, ADR_TEST_TYPES, TIR_TEST_TYPES all reference an array of IDS. Would you be happy for me to speak to our BA and put a story in the backlog to look at implementing the above for all of the test types?

Reply:
yeah so these IDs in the enum can have the const we talked about so
`IDS: [HVG_TEST_ID, BUS_TEST_ID]`
hope this make sense

Reply:
By doing this, it will make this test type enum inconsistent with all of the others. I think it should be kept consistent with all of the other enums. Perhaps, this could be addressed at a later date, to refactor them all.

Reply:
When you say "test type enum inconsistent with all of the others", do you mean in other repositories?
Also my suggestion doesn't change the enum. What I am suggesting is to put a string into a constant. Enum structure is intact so I don't understand how this will be inconsistent.
Thank you

Reply:
I'm talking about in the same enum file in this repo for example, HGV_TRL_ROADWORTHINESS_TEST_TYPES and ADR_TEST_TYPES etc many others are both structured in that way. I do understand that using the constants will make it more readable but I don't think that it's appropriate to be done as part of this story